Gopkg.toml

```diff
@@ -59,11 +75,36 @@
   name = "k8s.io/apiserver"
   branch = "master"
 [[override]]
-  name = "k8s.io/kube-aggregator"
```

Comment:
@pohly we are removing here and adding back in line 96, to make diff cleaner can you please keep this here and add  "k8s.io/apiextensions-apiserver" as a new [override]

Reply:
The entries are sorted now. I think that makes more sense than sticking to some historic ordering just to keep diffs smaller.

Reply:
agreeed